packages/edit-site/src/components/site-icon/style.scss

```diff
@@ -19,4 +19,13 @@
 	.edit-site-layout.is-full-canvas & {
 		border-radius: 0;
 	}
+
+}
+
+/* Apply focus border to both the site icon and image when the button is focused */
+button:focus {
+	.edit-site-site-icon__icon,
+	.edit-site-site-icon__image {
+		border: var(--wp-admin-border-width-focus) solid var(--wp-admin-theme-color);
```

Comment:
A border will change the sizing of the image. I believe we have a mixin or variable that uses the box-shadow for the focus style instead. Also, I think we should apply the focus style to the button, not the inner part of it.

Reply:
Thank you for your feedback @jeryj  When I try to add box-shadow I can see the outline on the right side and bottom only and cannot see it in top and left. This is because the Site Icon is on the Top Left. Please have a look at this screenshot 
![box_shadow](https://github.com/user-attachments/assets/a5e6659e-98da-468d-b373-606b2ae91641)

Please advice how to proceed in this case. Thanks.

Reply:
@jeryj if you have some time, please advice on this. Thanks in advance.